media/js/src/AssetDetail.jsx

```diff
@@ -136,35 +197,72 @@ export default class AssetDetail extends React.Component {
+                                                                <td span="0">
+                                                                    <div><label htmlFor="annotation-title">Selection Times</label></div>
+                                                                </td>
+                                                            </tr>
+                                                            <tr className="sherd-clipform-editing">
+                                                                <td>
+                                                                    <input
+                                                                        type="button" className="btn-primary"
+                                                                        onClick={this.onStartTimeClick}
+                                                                        readOnly value="Start Time" id="btnClipStart" />
+                                                                </td>
+                                                                <td width="10px">&nbsp;</td>
+                                                                <td>
+                                                                    <input
+                                                                        type="button" className="btn-primary"
+                                                                        onClick={this.onEndTimeClick}
+                                                                        readOnly value="End Time" id="btnClipEnd" /> </td>
+                                                                <td>&nbsp;
+                                                                </td>
                                                             </tr>
-                                                            <tr className="sherd-clipform-editing"><td><input type="text" className="timecode" id="clipStart" readOnly value="00:00:00" /><div className="helptext timecode">HH:MM:SS</div></td><td style={{width: '10px', textAlign: 'center'}}>-</td><td><input type="text" className="timecode" id="clipEnd" readOnly value="00:00:00" /><div className="helptext timecode">HH:MM:SS</div></td><td className="sherd-clipform-play"><input type="image" title="Play Clip" className="regButton videoplay" id="btnPlayClip" src="/media/img/icons/meth_video_play.png" /></td>
+                                                            <tr className="sherd-clipform-editing">
+                                                                <td>
+                                                                    <input
+                                                                        type="text" className="timecode" id="clipStart" readOnly
+                                                                        value={this.state.annotationStartTime} /><div className="helptext timecode">HH:MM:SS</div></td>
```

Comment:
the annotationStartTime and annotationEndTime are coming through as floats...likely these should be cast to int.

Reply:
Yeah, I definitely have some work to do there. I'll get that sorted out before a merge.